src/Log/LogCollection.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Deployer\Log;
+
+use Deployer\Collection\Collection;
+
+class LogCollection extends Collection
```

Comment:
It's not necessary, but I've seen that every function that is configurable from deployer is going to a collection, I just wanted to follow the same workflow you had.